### PR TITLE
Business dashboard theming - Card

### PR DESCRIPTION
**Pull request by dimafeldman**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6207

File: src/Themes/businessDashboard/components/Card/Header.st.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  -st-default: Header;
+}
+
+.root {
+  -st-extends: Header;
+  padding: 24px 24px 0;
+  align-items: flex-start;
+}
+
+.root::titleWrapper h3 {
```
Comment (shlomitc) [inline src/Themes/businessDashboard/components/Card/Header.st.css:12]: targeting the tag element like `h3` isn't a good idea. We always need to target the public apis as exposed in the stylesheet.

Can't you simply add margins to the `titleWrapper`? if so, then that's the easy solution.
If not, you'll need to do the following small change in the Card Header component: https://github.com/wix/wix-style-react/blob/77a62db25539472f397a4ce7b16738ac6e31c939/src/Card/Header/Header.js#L38
you'll need to add a className here:
```jsx
<Heading
  dataHook={DataHooks.title}
  className={classes.title}
  appearance="H3"
  children={title}
/>
```

an in the stylable file of it, just expose the right class
```css
:import {
  -st-from: "../../Heading/Heading.st.css"
  -st-default: Heading;
}
.title {
  -st-extends: Heading;
}
```

then change your selector to `.root::titleWrapper::title`

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: src/Themes/businessDashboard/components/Card/Header.st.css (hunk 1 above)
Comment (dimafeldman) [reply src/Themes/businessDashboard/components/Card/Header.st.css]: Thanks @shlomitc !
I added the className as you suggested

File: src/Themes/businessDashboard/test/businessDashboard.visual.js [hunk 2]
```diff
@@ -7,14 +7,19 @@ import { runTests as HeadingTests } from '../../../Heading/test/Heading.visual';
 import { runTests as TextTests } from '../../../Text/test/Text.visual';
 import { runTests as TextButtonTests } from '../../../TextButton/test/TextButton.visual';
 import { runTests as IconButtonTests } from '../../../IconButton/test/IconButton.visual';
+<<<<<<< HEAD
```
Comment (shlomitc) [inline src/Themes/businessDashboard/test/businessDashboard.visual.js:10]: looks like a wrong merge commit

Comment (dimafeldman) [reply src/Themes/businessDashboard/test/businessDashboard.visual.js:10]: oh wow! fixing. sorry

**Pull request merged**